modules/suvpar/Suvpar.py

```diff
@@ -48,7 +48,14 @@ def suvpar():
     df = df[df['ImageType'].str.contains("ORIGINAL")]
     # Consider only MR. Remove modalities such as PR and SR that are present in the original data.
     df = df[df.Modality == "MR"]
-
+    # Consider some Device Serial Number and remove other.
+    DSN = ['26356', '35386', '31130', '69618', '25992', '41546', '41563', '145596', '141301', '45988', '145384',
+     '141638', '000000000000GEMS',
```

Comment:
Instead of hard-coding these values, can you please accept and read a csv file called, "scanners.csv" with these values in it? (and commit a scanner.csv with dummy values such as 
#DeviceSerialNumber
scanner1
scanner2
)

Otherwise, we are making the code unnecessarily specific to the local deployment.

Reply:
got it